doc/building-running.md

```diff
@@ -0,0 +1,63 @@
+**Validated: 2020/03/26**
+
+# Building Cardano Node
+
+There are a number of ways the Cardano Node can be built and run, but the following document
+details the Nix and Ubuntu/Debian methods. The Nix method is probably more robust and foolproof
+than installing on Ubuntu/Debian.
+
+
+### Building under Nix
+
+The [Nix Package Manager][nix] can be installed on most Linux distributions by downloading and
+running the installation script:
+```
+curl https://nixos.org/nix/install > install-nix.sh
+./install-nix.sh
+```
+and following the directions.
+
+To improve build speed, it is possible to set up a binary cache maintained by IOHK (**this is
+optional**):
+```
+sudo mkdir -p /etc/nix
+cat <<EOF | sudo tee /etc/nix/nix.conf
+substituters = https://cache.nixos.org https://cache.nixos.org/ https://iohk.cachix.org https://hydra.iohk.io https://cache.nixos.org/
```

Comment:
`https://cache.nixos.org/` occurs three times on this line.

Reply:
Thanks for finding that. I wish that file format was a little more sane and easier ti eyeball for correctness.